docs/deploy/index.mdx

```diff
@@ -1,52 +1,26 @@
 ---
-title: Deployment overview
+title: Deployment Overview
 description: Deployment overview page
 sidebar_position: 0
 ---
 
-# Tacetve a gemit quisquam praestat meruisse gradientis
+# Deploying to Weaviate
```

Comment:
Wdyt about adding a small admonition to indicate that this section is under construction?